apps/desktop/src/components/v3/CommitRow.svelte

```diff
@@ -188,30 +136,11 @@
 	</div>
 </div>
 
-<ConflictResolutionConfirmModal
-	bind:this={conflictResolutionConfirmationModal}
-	onSubmit={editPatch}
-/>
-
-<CommitContextMenu
-	bind:menu={contextMenu}
-	{projectId}
-	leftClickTrigger={kebabMenuTrigger}
-	rightClickTrigger={commitRowElement}
-	{baseBranch}
-	{stackId}
-	{commit}
-	commitUrl={forge.current.commitUrl(commit.id)}
-	onUncommitClick={handleUncommit}
-	onEditMessageClick={openCommitMessageModal}
-	onToggle={(isOpen, isLeftClick) => {
-		if (isLeftClick) {
-			isOpenedByKebabButton = isOpen;
-		} else {
-			isOpenedByMouse = isOpen;
-		}
-	}}
-/>
+<ContextMenu bind:this={contextMenu} leftClickTrigger={kebabMenuTrigger}>
+	{#snippet menu(args)}
+		{@render menu2?.(args)}
```

Comment:
Rename `menu2` -> `menu`, and pass as prop rather than doing a `@render` inside the snippet.